### DISCUSSION TITLE
Missing icons for oh-my-zsh

### DISCUSSION THREAD

**Discussion by heseber** [tonsky/FiraCode#1285] [Ideas]
Oh-my-zsh uses a many icons for the prompt, e.g. an apple or penguin for the system you are on, branch symbols if you are in a git repository, etc.  These icons are provided, e.g., by the "MesloLGS-NF" font family. Would it be possible to add such icons to FireCode as well? Otherwise, the prompt in terminals in RStudio or VS Code looks ugly and less informative when using FiraCode instead of MesloLGS-NF (the latter does not have ligatures, though).

**Comment by mfocko** [comment]
There are patched fonts: https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/FiraCode
There's also https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/FiraMono if you don't want the ligatures.

They are not updated that regularly, but since there are not that many `*.ttf` variants of Fira Code, it is matter of 5-10 minutes of patching locally
